src/app/legacy/psammead/psammead-social-embed/src/index.test.jsx

```diff
@@ -128,19 +128,12 @@ describe('CanonicalSocialEmbed', () => {
       />
     );
     it('should render correctly for Twitter', async () => {
-      const { container, unmount } = render(twitterSocialEmbed);
+      const { container } = render(twitterSocialEmbed);
+
+      const button = screen.getByTestId('banner-button');
+      fireEvent.click(button);
+
       expect(container.firstChild).toMatchSnapshot();
-      expect(
-        document.querySelector(
-          'head script[src="https://platform.twitter.com/widgets.js"]',
-        ),
-      ).toBeTruthy();
-      unmount();
-      expect(
-        document.querySelector(
-          'head script[src="https://platform.twitter.com/widgets.js"]',
-        ),
-      ).toBeFalsy();
```

Comment:
Similar to the above: https://github.com/bbc/simorgh/pull/10562/commits/fb200acefdc1feaee78f34ae0605536ed56c52c1#diff-7432987f0acea427fd1311feb48d68d7f5aa675f3bb25f45c4e34db0d1e6275b